webapp-django/crashstats/manage/jinja2/manage/home.html

```diff
@@ -4,54 +4,86 @@
 
 {% block admin_title %}Admin Home{% endblock %}
 
+{% block site_css %}
+{{ super() }}
+{% stylesheet 'manage:home' %}
+{% endblock %}
+
 {% block mainbody %}
 
 <div class="panel">
     <div class="body">
+        <h1>Data</h1>
```

Comment:
Not `h1` please. It looks so wrong when the page header is "Admin Home" in a smaller font and suddenly there's these monster headings.